packages/myst-ext-button/tests/button.spec.ts

```diff
@@ -19,15 +19,61 @@ describe('Button component', () => {
     ]);
   });
 
-  it('should process button role without label correctly', () => {
-    const result = buttonRole.run({ name: 'button', body: 'http://example.com' }, new VFile());
+  it('should process autolink-style bodies', () => {
+    const result = buttonRole.run({ name: 'button', body: '<http://example.com>' }, new VFile());
     expect(result).toEqual([
       {
         type: 'link',
         class: 'button',
         url: 'http://example.com',
-        children: [],
+        children: [{ type: 'text', value: 'http://example.com' }],
+      },
+    ]);
+  });
+
+  it('should treat bare text (even if it looks like a URL) as a non-link button', () => {
```

Comment:
I feel like this test isn't needed because it's not really _intended_ behavior but just not behavior that's implemented yet